### DISCUSSION TITLE
$errors->any() always empty in Laravel v10.48.4

### DISCUSSION THREAD

**Discussion by Aibat1997** [laravel/framework#50886] [Q&A]
Hi, I validate some inputs and I can't display validation errors becouse $errors->any() always return empty array
![image](https://github.com/laravel/framework/assets/34008555/6c97f5a7-825b-40a7-8d50-8204717269a6)
![image](https://github.com/laravel/framework/assets/34008555/1d6d975b-5d2e-4483-9477-f291a2478c2d)
![image](https://github.com/laravel/framework/assets/34008555/595fe965-4835-440a-8679-e1995a33bb17)

but when I look  session it's return
![image](https://github.com/laravel/framework/assets/34008555/5a93378c-53f1-4b59-919d-bd38a2613335)
![image](https://github.com/laravel/framework/assets/34008555/a6c63e5e-a421-4897-a2a7-221401bcb519)

**Comment by ghost** [comment]
public function any()
    {
        return $this->count() > 0;
    }

use ->getBags()  or ->toArray() and look at the array structure.


This is what happens when people don't put return types and types parameters when codding...

**Reply by henzeb** [reply]
This is also what happens when people don't read code before asking questions ...

**Reply by ghost** [reply]
I think he asked the wrong question. The ->any is used ok, but the error printing does not work.

**Reply by Aibat1997** [reply]
Thanks for the answers, yes, the thing is that validation errors are not displayed in my blade template, $errors->any() always returns an empty array, although when I made dd(request()->session()->get('errors')) it displays a list of errors

**Reply by ghost** [reply]
$errors->any()  returns bool not empty array.

dd($errors->getBags()) or ->toArray() to see the format so you can print the messages.

**Reply by Aibat1997** [reply]
dd($errors->getBags()) it returns this

![image](https://github.com/laravel/framework/assets/34008555/2c8160a7-bd1f-46ec-a8d8-26bacc59f15d)

**Reply by ghost** [reply]
So getBags() returns array. so 
```

foreach ((($errors->getBags()['default'] ?? null)?->getMessages() ?? []) as $col => $messages) {
   echo \implode(',', $messages);
}
```

**Reply by Aibat1997** [reply]
Thanks, but why doesn't the code work as described in the documentation? I did everything as described in the documentation

**Reply by ghost** [reply]
Because the code is the best documentation to rely on.
I updated my comment above.

**Reply by Aibat1997** [reply]
Everything works correctly in v10.11.0, but for some reason it doesn't work in this version.

**Reply by ghost** [reply]
I suggest then to edit your question or create a new one and explain exactly that. Leave out the ->any() part.

**Reply by Aibat1997** [reply]
After submitting the form, the page just reloads, but without displaying validation errors. But when I output dd($errors->any()) it shows true, also when I output dd($errors->all()) it shows an array of validation errors. I can't figure out what the problem is

**Reply by Aibat1997** [reply]
SESSION_DRIVER=file
SESSION_LIFETIME=120

**Reply by ghost** [reply]
pls put here dd($erros->all());

**Reply by Aibat1997** [reply]
everything was working properly, it was my fault. Turns out there was a js library that updated the page covertly
